price-server/src/exchange_price_cache.rs

```diff
@@ -95,4 +100,61 @@ impl ExchangePriceCacheInner {
         }
         Err(ExchangePriceCacheError::NoPriceAvailable)
     }
+
+    fn mid_price_of_one_sat(&self) -> Result<UsdCents, ExchangePriceCacheError> {
```

Comment:
What you've done here is correct - following the existing pattern...

however I think we should cleanup the `ExchangePriceCache`.

It should just expose the `BtcSatTick` directly:
```
impl ExchangePriceCache {
  pub fn get_latest_tick()
}
```

Then in the us case you can call:
```
cache.get_latest_tick().mid_price_of_one_sat()
```

then we don't have to duplicate all the use case functions 3 times.